disperser/common/v2/blob.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 
 	pb "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2"
-	core "github.com/Layr-Labs/eigenda/core/v2"
+	corev2 "github.com/Layr-Labs/eigenda/core/v2"
```

Comment:
FYI this fixes swagger docs to resolve `v2.BlobHeader` (previously it was resolving v1 `core.BlobHeader`)

Reply:
Moved to separate PR https://github.com/Layr-Labs/eigenda/pull/1199/files